src/formats/ibm_n88basic.textpb

```diff
@@ -0,0 +1,117 @@
+comment: 'N88-BASIC 5.25"/3.5" 77-track 26-sector DSHD'
+
+flux_sink {
+	drive {
+		high_density: true
+	}
+}
+
+flux_source {
+	drive {
+		high_density: true
+	}
+}
+
+image_reader {
+	filename: "ibm_n88basic.img"
+	img {
+		tracks: 77
+		sides: 2
+		trackdata {
+            track: 0
+            up_to_track: 0
+			sector_size: 128
+			sector_range {
+				start_sector: 1
+				sector_count: 26
+			}
+		}
+        trackdata {
+            track: 0
```

Comment:
This range overlaps the previous trackdata clause, and matching clauses are merged top to bottom, so the `sector_size: 256` will override the 128 in the clause above.

Reply:
Indeed, reordered the other block as well and added an imagewriter definition.

Reply:
Yup, LGTM.

BTW, do you have actual hardware and have verified this works? In which case this could very plausibly get a dinosaur in the big table in the README...

Reply:
I have real hardware and confirmed that .d88 files read and written with this layout work, however I have not tested this .textpb file specifically.